pkg/deploy/spindeploy/transformer/accounts_test.go

```diff
@@ -0,0 +1,39 @@
+package transformer
+
+import (
+	"context"
+	"github.com/armory/spinnaker-operator/pkg/apis/spinnaker/v1alpha2"
+	"github.com/armory/spinnaker-operator/pkg/inspect"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestAddSpringProfile(t *testing.T) {
+	c := v1alpha2.SpinnakerConfig{
+		Profiles: make(map[string]v1alpha2.FreeForm),
+	}
+	if assert.Nil(t, addSpringProfile(&c, "clouddriver", "accounts")) {
```

Comment:
Maybe you could use the constant defined in accounts module here (and in the assertion)

Reply:
It's just a test value. Replaced with `test` to avoid confusion.